vortex-serde/src/layouts/read/footer.rs

```diff
@@ -35,21 +36,22 @@ use crate::FLATBUFFER_SIZE_LENGTH;
 /// │    Magic bytes (4 bytes)   │
 /// └────────────────────────────┘
```

Comment:
update this

Reply:
Not to be super pedantic, but: when we say "Footer", do we really mean Schema + Layouts + row_count (and anything else we may want to add later)? Where we currently don't but will eventually support footers that don't fit in the initial read.

The Footer is then followed by a Postscript, which is of a compile-time known fixed size (given Version from the EOF block). The Postscript can/should be assumed to forever-and-always be read in the initial read (since we'll encode both of those sizes as compile-time constants).

Currently, the Postscript contains offsets for the Schema and the... Sub-Footer? (called "Footer" in fbs)... that contains Layout and row_count.

That last part implies to me that the fbs Footer is maybe slightly wrong, naming-wise? though I know why it is the way that it is

Reply:
the in memory struct is likely wrongly named

Reply:
cool, so neither the metadata tables nor the schema are part of what we call the "Footer"?

i.e., top-level file looks like:

1. Data array IPC messages
2. Metadata "table" array IPC messages
3. Schema
4. Footer
    a. Layout
    b. Row count
5. Postscript (32 bytes, can change if we increment version in EOF)
    a. Schema Offset
    b. Footer Offset
6. EOF (8 bytes)
    a. 2 bytes of padding
    b. u16 version
    c. 4 bytes of magic